compiler/rustc_mir_dataflow/src/storage.rs

```diff
@@ -1,11 +1,12 @@
-use rustc_index::bit_set::BitSet;
+use rustc_index::bit_set::GrowableBitSet;
 use rustc_middle::mir::{self, Local};
 
 /// The set of locals in a MIR body that do not have `StorageLive`/`StorageDead` annotations.
 ///
 /// These locals have fixed storage for the duration of the body.
```

Comment:
The comment should explain the unconventional signature for this function, and what the caller is supposed to pass for the `locals` argument.